unit_tests/UnitTestActuatorFASTGaussian.C

```diff
@@ -0,0 +1,63 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2014 Sandia Corporation.                                    */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+/*
+ * UnitTestActuatorDiskFAST.C
+ *
+ *  Created on: Nov, 2019
+ *      Author: tony
+ */
+
```

Comment:
This information is available in git history and `git-blame`, we haven't been putting individual names in files.